.husky/commit-msg

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+npx --no-install commitlint --edit $1
```

Comment:
```suggestion
yarn commitlint --edit $1
```

You have @commitlint/cli in your devDependencies so you can call the local binary with `yarn <binary name>`